dev-app/src/screens/MerchantSelectScreen.tsx

```diff
@@ -109,13 +130,22 @@ export default function MerchantSelectScreen() {
           placeholder="sk_test_xxx"
           editable={!isAddPending}
         />
-        <ListItem
-          color={colors.blue}
-          title="Add Merchant"
-          onPress={onAddAccount}
-          disabled={isAddPending}
```

Comment:
can we keep this above Direct Payment field, so it's clear the button is not related to Direct Payment input

Reply:
Okey.